src/app/TempConverter/TempConverter.c

```diff
@@ -0,0 +1,45 @@
+#include "TempConverter.h"
+
+float TempConverter(float resistance) {
+    float temp[] = {953773.550, 886261.171, 823958.701, 766433.156, 713289.955, 664169.306, 618742.960, 576711.277,

```

Comment:
This should be outside the function and have the static keyword. As it is, this program will need to copy every value into this array from program memory (essentially the hard drive of the micro) to the stack (RAM) every time the function is called. Putting this outside the function makes the program initialize it in RAM before beginning the main function only once.